app/models/UserProfile.ts

```diff
@@ -15,6 +15,16 @@ export default class UserProfile extends BaseModel {
     @Column({ nullable: true })
     public dob: Date;
 
+    /**
+     * Gender is specified as a single 4-bit character, with the range of 26 possible options with
+     * a total of 52 if being case insensitive. This will cover them for the base M and F with
+     * the option to support a larger range of inclusivity if possible..
+     *
+     * Null support for if the user does not want to provide there gender during creation.
+     */
+    @Column({ nullable: true, length: 1 })
+    public gender: string;
```

Comment:
Convert type to enum and rename the column from `gender` to `sex` to ensure we are referring to the biological terminology.